css/style.css

```diff
@@ -0,0 +1,234 @@
+body {
+  background-color: #eee;
+  display: flex;
+  flex-direction: column;
+  font-family: sans-serif;
+  color: #000;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  background-color: #eee;
+  display: grid;
+  grid-template-columns: 300px 1fr;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  padding-top: 10px;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  max-width: 750px;
+  position: relative;
+  top: 40px;
+  z-index: 0;
+}
+
+#header-img {
+  grid-column: 1;
+  width: 100%;
+  align-self: center;
+}
+
+#nav-bar {
+  grid-column: 2;
+  align-self: center;
+  justify-self: right;
+  display: flex;
+  margin-right: 5px;
+}
+
+.nav-link {
+  text-decoration: inherit;
+  color: inherit;
+  display: inline-block;
+  margin: 5px;
+}
+
+/* input0[type=submit] {
+      text-align: center;
+  } */
+
+#intro {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#intro h2 {
+  margin-bottom: 15px;
+}
+
+#form {
+  display: grid;
+  align-items: center;
+  grid-template-rows: 1fr 1fr;
+  grid-gap: 10px;
+  margin-bottom: 30px;
+}
+
+input[type="submit"] {
+  width: 200px;
+  background-color: #4caf50;
+  padding: 0 32px;
+  border-style: none;
+  border-radius: 3px;
+  color: #fff;
+}
+#features {
+  margin-top: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
```

Comment:
I think that jumping one line before writing the next CSS rule is good for code clearness and readability.